EIPS/eip-6811.md

```diff
@@ -0,0 +1,150 @@
+---
+eip: 6811
```

Comment:
The EIP number will be added according to the PR number, you can leave it empty and add it later